### PR TITLE
Added PHP73

### PR DESCRIPTION
**Pull request by phracek**
This pull request adds support for PHP 73. #https://github.com/sclorg/s2i-php-container/issues/249

### CODE REVIEW

**Comment by pkubatrh** [comment]
Will need to be rebased to pick up https://github.com/sclorg/s2i-php-container/commit/2f7fd0f6c4480a604c3622ae5092cb714a958a18

**Comment by phracek** [comment]
@pkubatrh PR is rebased with the latest `container-common-scripts`. Please take a look on it. Thanks.

**Comment by pkubatrh** [comment]
[test-openshift]

File: imagestreams/php-rhel7.json [hunk 1]
```diff
@@ -47,6 +47,26 @@
...
+          "description": "Build and run PHP 7.3 applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-php-container/blob/master/7.3/README.md.",
+          "iconClass": "icon-php",
+          "tags": "builder,php",
+          "supports":"php:7.3,php",
+          "version": "7.3",
+          "sampleRepo": "https://github.com/sclorg/cakephp-ex.git"
+        },
+        "from": {
+          "kind": "DockerImage",
+          "name": "registry.redhat.io/rhscl/php-72-rhel7:latest"
```
Comment (pkubatrh) [inline imagestreams/php-rhel7.json]: `php-73-rhel7` here, same for the other imagestreams files

File: 7.2/s2i/bin/assemble [hunk 2]
```diff
@@ -12,11 +12,6 @@ mv /tmp/src/* ./
 fix-permissions ./
 fix-permissions ${HTTPD_CONFIGURATION_PATH}
 
-# Change the npm registry mirror if provided
-if [ -n "$NPM_MIRROR" ]; then
-	npm config set registry $NPM_MIRROR
-fi
-
```
Comment (pkubatrh) [inline 7.2/s2i/bin/assemble]: I think this is a rebase error right?

File: 7.2/test/run [hunk 3]
```diff
@@ -36,7 +36,7 @@ container_ip() {
 }
 
 run_s2i_build() {
-  ct_s2i_build_as_df file://${test_dir}/test-app ${IMAGE_NAME} ${IMAGE_NAME}-testapp ${s2i_args} $(ct_build_s2i_npm_variables)
+  ct_s2i_build_as_df file://${test_dir}/test-app ${IMAGE_NAME} ${IMAGE_NAME}-testapp ${s2i_args}
```
Comment (pkubatrh) [inline 7.2/test/run]: same here

File: 7.2/test/run [hunk 4]
```diff
@@ -198,7 +198,7 @@ test_ssl() {
 }
 
 test_ssl_own_cert() {
-  ct_s2i_build_as_df file://${test_dir}/self-signed-ssl ${IMAGE_NAME} ${IMAGE_NAME}-test-self-signed-ssl ${s2i_args} $(ct_build_s2i_npm_variables)
+  ct_s2i_build_as_df file://${test_dir}/self-signed-ssl ${IMAGE_NAME} ${IMAGE_NAME}-test-self-signed-ssl ${s2i_args}
```
Comment (pkubatrh) [inline 7.2/test/run]: same here

File: 7.2/test/run-openshift [hunk 5]
```diff
@@ -1 +0,0 @@
-../../test/run-openshift
```
Comment (pkubatrh) [inline 7.2/test/run-openshift]: We want to keep the symlink here

Comment (pkubatrh) [reply 7.2/test/run-openshift]: Same for all the `../../test/*` symlinks actually.

File: 7.2/s2i/bin/assemble (hunk 2 above)
Comment (phracek) [reply 7.2/s2i/bin/assemble]: yes

**Comment by phracek** [comment]
Good catch during the review. I have to be careful during next php or another release.

**Comment by pkubatrh** [comment]
[test-openshift]

**Comment by pkubatrh** [comment]
Lets wait for the CI to finish before merging.

@phracek Do you plan to follow-up with PRs to enable Fedora and Centos/rhel7 builds?

**Comment by pkubatrh** [comment]
CI is green, so merging.

**Comment by xrow** [comment]
Just as a hint the image is missing at the moment. Maybe you  know about this.

pull centos/php-73-centos7

**Pull request merged**